backstage-plugin/plugins/open-dora/src/models/DfBenchmarkData.ts

```diff
@@ -0,0 +1,10 @@
+export interface dfBenchmarkData {
+  key: dfEnum;
+}
+
+export enum dfEnum {
```

Comment:
I think `enum` would be useful if you were referring to the values by their alias eg `dfEnum.LT6Month` but we don't do that anywhere. Instead, I think a simple composite type is better:
```ts
export type dfBenchmarkKey = 'on-demand' | 'week-month' | 'month-6month' | 'lt-6month';
```